google/cloud/bigtable/mutations.h

```diff
@@ -59,6 +49,36 @@ Mutation SetCell(std::string family, std::string column, std::string value);
  */
 constexpr std::int64_t ServerSetTimestamp() { return -1; }
 
+/// Create a mutation to set a cell value.
+template <typename ColumnType, typename ValueType>
+Mutation SetCell(std::string family, ColumnType column,
+                 std::chrono::milliseconds timestamp, ValueType value) {
```

Comment:
I think we want forwarding references here too. Basically, I think anywhere we use `std::forward` type argument template type should be like `T&&`. And vice versa.

Reply:
Done.